man/memkind.3

```diff
@@ -524,6 +530,41 @@ section if it is not.
 .BR MEMKIND_PMEM_MIN_SIZE
 The minimum size which allows to limit the file-backed memory partition.
 .sp
+.B "STATISTICS:"
+.br
+The functions described in this section define a way to get specific memory allocation statistics.
+.PP
+.BR memkind_update_cached_stats ()
+is used to force an update of cached dynamic allocator statistics.
+Statistics are periodically updated by memkind library and this method allows to force update it on demand.
+.PP
+.BR memkind_get_stat ()
+gets from specified
+.I kind
+specific type of statistic
+.I stat
+and returns it in
+.IR "value".
+For more details about
+.IR stat
+see the
+.B "MEMORY STATISTICS TYPE"
+section below.
+If
+.I kind
+is
+.IR "NULL" ,
+measured statistic applies to memory used by the whole memkind library.
+.BR Note:
+Statistics are periodically updated by memkind library. That is why when calling a
```

Comment:
You need to call update because statistics are *not* updated real-time (or, "are cached").

What you wrote implies why you might need to update stats (because periodically means not always), but it's confusing IMHO.